src/index.jsx

```diff
@@ -8,8 +8,4 @@ import App from './App';
 import './style.css';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
```

Comment:
I think we can keep `React.StrictMode` to the build. Think, it'll be useful to have it check for any unexpected side-effects during development. Shouldn't affect production.

Reply:
Woops forgot I removed that. I'll add it back in.

Reply:
Fixed